src/tags/tags.toml

```
@@ -59,6 +59,7 @@ content = """
 - [**Archangel** **¹** **²** **⁴** **⁵**](<https://github.com/favware/archangel>)
 - [**Arima** **¹** **²** **⁴**](<https://github.com/arimajs/Arima>)
 - [**Dragonite** **¹** **²** **⁴** **⁵**](<https://github.com/favware/dragonite>)
+- [**Radon** **2** **4**](https://github.com/EvolutionX-10/Radon)
```

```suggestion
- [**Radon** **3** **4**](https://github.com/EvolutionX-10/Radon)
```

- Your bot doesn't use ESM as it doesn't have `"type": "module"` in the package.json
- Due to the kinda hard to read code (lack of blank lines, easily readable formatted code, and some magic numbers here and there) I'd rather this be marked as advanced

---

Sure, will consider your suggestion!